tests/test_dashboards.py

```diff
@@ -16,6 +16,10 @@ def setUpClass(cls) -> None:
     def tearDownClass(cls) -> None:
         id_ = cls.api.get_dashboard_id('pylana-dashboard')
         cls.api.delete_dashboard(id_)
+
+        test_dashboard_id_ = cls.api.get_dashboard_id('incident-test-dashboard')
+        cls.api.unshare_dashboard(test_dashboard_id_,
+                                  [cls.api.user.organization_id])
```

Comment:
We should denote discarded returns with `_ = ...`. Otherwise the line could be an inplace modification of whatever is behind `cls.api`. It can still be with the discarding, but it make it much clearer, that we expect a return value.